### PR TITLE
Run tests against examples in Markdown files

### PR DESCRIPTION
**Pull request by bahmutov**
Here is a first second stab at directly testing examples in our API commands Markdown files. I have added `@cypress/fiddle` and creates a command that points at the Markdown source pages. I have surrounded a few examples with fiddle block comments, so they are invisible in "normal" processing, but Cypress can run these files as test files

Screen Shot 2019-10-21 at 2 46 15 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,6 +92,7 @@
     "chalk": "2.4.2",
     "cheerio": "1.0.0-rc.3",
     "coffee-react": "5.0.1",
+    "coffeescript": "2.4.1",
```
Comment (bahmutov) [inline package.json]: for some reason, `coffeescript` dependency was not found by default, even if it was a dependency of `@cypress/browserify-preprocessor`

**Comment by bahmutov** [comment]
@jennifer-shehane here is an example of testing command examples

File: package.json (hunk 1 above)
Comment (jennifer-shehane) [reply package.json]: I'm confused why we need this. There is no coffeescript in this repo at all. This a dependency of @cypress/browserify-preprocessor??

**Comment by jennifer-shehane** [comment]
This does require maintenance. 

- I assume these tests will run in CI - running each time we push code - which requires supporting and documenting.
- Every js and html snippet now has these comments wrapping them - which requires supporting and documenting.

I understand - this may be nice, but this is extremely low priority compared to any other feature we could spend time developing in the Test Runner.

**Pull request closed without merging**